migrating_to_2.0/general.rst

```diff
@@ -54,6 +54,15 @@ Extensions
 
 Work in progress
 
+Hooks
+^^^^^
+
+- Hooks folder has been updated to ``~/.conan2.extensions/hook``;
```

Comment:
is it really `.conan2.extension/hook`, not `.conan2/extensions/hooks`?

Reply:
```suggestion
- Hooks folder has been updated to ``~/.conan2/extensions/hook``;
```

Reply:
Good catch!